crates/agent-sql/src/directives/beta_onboard.rs

```diff
@@ -41,39 +41,86 @@ pub async fn tenant_exists(
     Ok(exists.is_some())
 }
 
-/*
-TODO(johnny): A fast-follow will be inserting `ops/` specifications for the user:
-insert into live_specs (catalog_name, last_build_id, last_pub_id, spec, spec_type) values
-    ('ops/' || $2 || 'logs',  '00:00:00:00:00:00:00:00', '00:00:00:00:00:00:00:00', $4, 'collection'),
-    ('ops/' || $2 || 'stats', '00:00:00:00:00:00:00:00', '00:00:00:00:00:00:00:00', $5, 'collection')
-*/
-pub async fn provision_user(
-    user_id: Uuid,
-    tenant: &str,
+// ProvisionedTenant is the shape of a provisioned tenant.
+#[derive(Debug)]
+pub struct ProvisionedTenant {
+    // Draft into which provisioned catalog specs should be placed.
+    // It will be queued for publication upon the commit of this transaction.
+    pub draft_id: Id,
+}
+
+pub async fn provision_tenant(
+    accounts_user_email: &str,
```

Comment:
Why email here, but `tenant_user_id` down below?

Reply:
accounts_user_email is, for example, `accounts@estuary.dev`. It's a separate account from the user ID on whose behalf we're acting.